sdks/python/build-requirements.txt

```diff
@@ -21,6 +21,9 @@ setuptools
 wheel>=0.36.0
 
 grpcio-tools==1.37.0
+# TODO(https://github.com/apache/beam/issues/23734): the sdist for grpcio==1.50.0 is failing on GH workers
+# pin grpcio to the previous version.
+grpcio==1.49.1
```

Comment:
Should we pick the same version for grpcio and grpcio-tools? (We should probably do it at the beginning...)

Reply:
What would be the consequence if the version between the both differ? 

IIUC, grpcio-tools==1.37.0 has a requirement of grpcio>=1.37.0. I think having a version equal to or greater than grpcio-tools for grpcio would be fine.

Reply:
can we do this pinning for the mac platform only? Thanks!

Reply:
Done